mne/viz/_brain/mplcanvas.py

```diff
@@ -11,35 +11,41 @@
 class MplCanvas(object):
     """Ultimately, this is a QWidget (as well as a FigureCanvasAgg, etc.)."""
 
-    def __init__(self, brain, width, height, dpi):
-        from PyQt5 import QtWidgets
+    def __init__(self, brain, width, height, dpi, notebook=False):
+        import matplotlib.pyplot as plt
```

Comment:
importing `plt` imports/initializes whatever the default backend is so this is a regression of sorts. Probably better to select FigureCanvasQtAgg or whatever the notebook equivalent is